src/PolyhedralGeometry/PolyhedralFan/properties.jl

```diff
@@ -362,41 +362,17 @@ Return the primitive collections of a polyhedral fan.
 
 # Examples
 ```jldoctest
-julia> primitive_collections(normal_fan(Oscar.simplex(3)))
+julia> primitive_collections(normal_fan(simplex(3)))
 1-element Vector{Vector{Int64}}:
  [1, 2, 3, 4]
 ```
 """
 function primitive_collections(PF::PolyhedralFan)
-    # collect data
-    cones = [findall(x->x!=0, l) for l in eachrow(pm_object(PF).MAXIMAL_CONES)]
-    all_points = [i for i in 1 : pm_object(PF).N_RAYS]
-    d_max = maximum([length(i) for i in cones]) + 1
-    # identify and return the primitive collections
-    collections = Vector{Int}[]
-    for d in 1:d_max
-        checked  = Vector{Int}[]
-        for cone in cones
-            d <= length(cone) || continue
-            for I_minus_j in Oscar.Hecke.subsets(cone, d)
-                scanner = setdiff(all_points, I_minus_j)
-                for j in scanner
-                    I = vcat(I_minus_j, [j])
-                    I in checked && continue
-                    push!(checked, I)
-                    # (1) I is contained in the primitive collections iff it is not contained in any cone
-                    if !any(test_cone -> issubset(I, test_cone), cones)
-                        # (2) I is generator of the primitive collections iff primitive_collections does not contain a "smaller" generator
-                        if !any(prim -> issubset(prim, I), collections)
-                            push!(collections, I) # add new generator
-                        end
-                    end
-                end
-            end
-        end
-    end
-    # return the computed primitive collections
-    return collections
+    simplicial = pm_object(PF).SIMPLICIAL::Bool
```

Comment:
I wanted to suggest to change this to
```suggestion
    simplicial = issimplicial(PF)
```
but then saw that `issimplicial` is only implemented for `PolyhedralComplex`, not `PolyhedralFan`... but why?

Reply:
Good point @fingolfin. This lead me to wonder more generally which properties are currently supported for fans and if we should/could directly use to compute properties of toric varieties rather than making direct calls to `Polymake`. This is orthogonal (or at least more general) than the discussion here, so I opened a separate issue: https://github.com/oscar-system/Oscar.jl/issues/1055

Reply:
Come on guys, it is not there because it is not implemented yet -- like many other features we want for Oscar. This is a draft and I am still working on it.